src/components/scene-sound.js

```diff
@@ -0,0 +1,12 @@
+AFRAME.registerComponent("scene-sound", {
+  multiple: true,
+  schema: {
+    sound: { type: "string" },
+    on: { type: "string" }
+  },
+
+  init() {
+    const sound = this.el.components[`${this.attrName.replace("scene-", "")}`];
```

Comment:
i'm assuming we didn't write the `sound` behavior? because it seems like this would be a natural extension to that

Reply:
Yea, the `sound` component is in `aframe.` Actually, I may need to modify it anyway because I need it not to spam the log with warnings whenever the app tries to rapidly play the same clip such that it exceeds the sound's poolSize (e.g. this happens when I scroll with the mouse wheel). The last attempts are rightly ignored, except for this warning which I don't want to have happen because I expect this to happen and am OK with it.